templates/voting_platform/_layout_base.html.twig

```diff
@@ -25,7 +25,23 @@
     <header class="voting-platform__header voting-platform-theme__{{ designation_type }}">
         <div class="l__wrapper--wide l__row l__row--h-stretch l__row--wrap">
             <div class="l__row space--20-0">
-                <a href="{{ path('homepage') }}" class="link--no-decor text--logo b__nudge--right-large">EM!</a>
+                {% if is_renaissance_host %}
+                    <a href="{{ path('app_renaissance_homepage') }}" class="link--no-decor text--logo b__nudge--right-large">
+                        <svg style="width: auto; height: 20px; filter: invert(32%) sepia(53%) saturate(2638%) hue-rotate(176deg) brightness(103%) contrast(89%);" fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 296 26">
+                            <g clip-path="url(#a)" fill="currentColor">
+                                <path d="M0 .432h8.357c5.906 0 9.184 2.625 9.184 7.803 0 4.351-2.305 6.834-6.554 7.553l6.843 9.818h-2.16l-6.7-9.637H1.73v9.637H0V.432Zm8.86 13.81c4.322 0 6.843-1.907 6.843-6.007 0-3.955-2.521-6.077-6.843-6.077H1.73v12.083h7.13ZM28.503.432h16.028v1.726h-14.3v9.745h13.616v1.727H30.232v10.248h14.3v1.726h-16.03V.432Zm25.879 0h2.81l14.334 20.462V.432h2.773v25.172h-2.846L57.12 5.214v20.392h-2.736V.432h-.002Zm38.186 0h3.098l9.831 25.172h-2.954l-2.305-5.79H87.922l-2.233 5.79h-2.99L92.568.432Zm6.664 16.757L94.081 3.955 89.003 17.19h10.23ZM113.898.432h2.736v25.172h-2.736V.432Zm12.146 20.748 2.916-2.947c1.549 2.013 3.639 3.883 7.673 3.883 3.204 0 4.79-1.51 4.79-3.416 0-1.618-1.007-3.02-3.961-3.525l-3.063-.503c-4.61-.755-7.383-3.344-7.383-7.444 0-4.495 4.107-7.228 8.897-7.228 4.465 0 7.312 1.906 9.328 4.028l-2.989 3.164c-1.405-1.69-3.423-3.308-6.519-3.308-3.098 0-4.395 1.726-4.395 3.237 0 1.69 1.08 2.877 3.999 3.38l2.52.431c5.403.935 7.888 3.705 7.888 7.696 0 4.244-3.565 7.373-9.652 7.373-4.683-.001-7.96-2.194-10.049-4.82Zm28.094 0 2.916-2.947c1.549 2.013 3.639 3.883 7.673 3.883 3.204 0 4.79-1.51 4.79-3.416 0-1.618-1.009-3.02-3.961-3.525l-3.063-.503c-4.61-.755-7.383-3.344-7.383-7.444 0-4.495 4.107-7.228 8.897-7.228 4.465 0 7.311 1.906 9.328 4.028l-2.989 3.164c-1.405-1.69-3.423-3.308-6.519-3.308-3.098 0-4.395 1.726-4.395 3.237 0 1.69 1.08 2.877 3.999 3.38l2.52.431c5.402.935 7.887 3.705 7.887 7.696 0 4.244-3.565 7.373-9.651 7.373-4.683-.001-7.961-2.194-10.049-4.82ZM190.443.432h5.114l9.869 25.172h-5.548l-2.016-4.962H188.1l-1.981 4.962h-5.547L190.443.432Zm5.69 15.463-3.134-8.199-3.134 8.2h6.268ZM219.692.432l10.193 17.046V.432h5.329v25.172h-6.411L218.611 8.631v16.973h-5.367V.432h6.448Zm24.558 12.55C244.25 5.645 249.761 0 257.181 0c4.574 0 7.635 1.762 9.977 4.495l-4.142 4.243c-1.405-1.619-2.666-2.806-5.51-2.806-3.601 0-6.483 2.986-6.483 7.049 0 4.136 2.809 7.084 6.555 7.084 2.989 0 4.538-1.258 6.159-3.489l3.962 4.388c-2.342 3.129-5.907 5.035-10.662 5.035-7.455.001-12.787-5.608-12.787-13.017ZM275.323.432h19.017v5.754h-12.462v3.812h11.848v5.538h-11.848v4.314h12.462v5.754h-19.017V.432Z"/>
```

Comment:
Could you replace this SVG code by `{% include 'renaissance/partials/logo-text.svg.twig'` please?